src/components/DevDummyLogin/index.js

```diff
@@ -62,12 +71,16 @@ const S = StyleSheet.create({
 const DevDummyLogin = ({resetClickTime = () => {}}) => {
   const {ENABLE_DEV_ONLY_FEATURE} = configEnv;
   const {setAuth} = useUserAuthHook();
+  const [dummyInput, setDummyInput] = React.useState('');
 
   const [dummyUsers] = React.useState([
     {name: 'fajarismv2', humanId: 'fajarismv2'},
     {name: 'halofajarism', humanId: 'halofajarism'},
     {name: 'agita', humanId: 'agita'},
     {name: 'usup', humanId: 'P19FGPQGMSZ5VSHA0YSQ'},
+    {name: 'new_test_account1', humanId: 'newtestaccount1'},
+    {name: 'new_test_account2', humanId: 'newtestaccount2'},
+    {name: 'new_test_account3', humanId: 'newtestaccount3'},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's not a critical issue, hardcoding test account data directly in the component might not be the best practice. Consider moving this data to a separate configuration or data file. This would make the code cleaner and easier to maintain.



<!-- This is an auto-generated comment by OSS CodeRabbit -->